### ISSUE TITLE
Feature request : ticketing integration

### ISSUE THREAD

**Issue by Teninx** [amidaware/tacticalrmm#1710]
Does it possible to had integration with Gestsup?

**Comment by compgeniuses** [comment]
i think the BEst integration would actually be [GLPi](https://glpi-project.org/)

And the agents could also integrate with glpi inventory agents.

would be interesting

**Comment by dinger1986** [comment]
yes I have looked at GLPi, also we need them as project to be interested in doing an integration

**Comment by compgeniuses** [comment]
@dinger1986 that was way too fast a response, interesting.

So let me try to Pork them at their Github Page, but i think an official communication to their contact page would be the best approach. considering its also an opensource project,

PR are also accepted, and they are quite receptive bunch.

**Comment by dinger1986** [comment]
see what they say, would be nice to do something with another project but right now, we have so many other features etc we are working on which takes priority from the developers for us but could work on something together.

**Comment by compgeniuses** [comment]
I have posted on the matter on the USer Echo page here: https://glpi.userecho.com/communities/1/topics/2308-remote-monitering-and-management-integration-tactical-rmm

**Comment by g-bougard** [comment]
Hello there,

a glpi-agent integration has been requested there: glpi-project/glpi-agent#700

I started to develop a module but actually it should only work on windows. The module idea is to only recover the `AgentId`. I would be happy to add linux & MacOS support if people can help explaining how to recover this data on that cases.

In GLPI, we should then be able to create a dedicated link on the relevant asset.
